packages/cascara/src/specs/Table/Table.spec.mdx

```diff
@@ -1,7 +1,155 @@
 ---
 title: Table
+status: Draft
+
 ---
 
 # Table
 
-Let's see if this shows up at our application root.
+## Table Types
+
+- Data table The standard table layout with a table header and rows of data
+- Grid view - more of a gallery view. No table header. Could be considered a list. Need to discuss if we want to make this a separate component or a variation on a table.
```

Comment:
Hi @CaffeinatedAllie, from the technical perspective the Grid implementation would be very similar to Table but having it separate might make more sense. Maintaining both together would be hard.

@brianespinosa what do you think?

Reply:
Would it be better to describe this as a "list" item.... and when we get to lists we can have multiple formats and grid/gallery view could be one??

Reply:
@lordpixel hard to say yet. Initially, yes I think it should be a separate component for managing that different display experience. The public API pertaining to the data input and configuration might be the same, but other display props might be different that control the experience. If the input props are 1:1 then we could have the overall admin layout experience later on where you could switch between display views if you want. But the implementation of that might be left to the developer initially.

@CaffeinatedAllie I think it is good we are thinking about that.

IMO, let's define that as a separate component for now and we can evaluate working it into an overall layout where switching views "just works" later.